android/src/main/kotlin/com/whelksoft/flutter_native_timezone/FlutterNativeTimezonePlugin.kt

```diff
@@ -9,45 +10,58 @@ import io.flutter.plugin.common.MethodChannel
 import io.flutter.plugin.common.MethodChannel.MethodCallHandler
 import io.flutter.plugin.common.MethodChannel.Result
 import io.flutter.plugin.common.PluginRegistry.Registrar
+import java.util.*
+import kotlin.collections.ArrayList
 
-class FlutterNativeTimezonePlugin: FlutterPlugin, MethodCallHandler {
+class FlutterNativeTimezonePlugin : FlutterPlugin, MethodCallHandler {
 
-  private lateinit var channel : MethodChannel
+    private lateinit var channel: MethodChannel
```

Comment:
The key change in this file LGTM.

(Although I suspect it could get reviewed and merged faster if you leave every line unchanged except for the key lines you substantively changed, because at the moment every line shows up as a change in the diff and it's hard to see which parts you actually changed when reviewing it. Maybe @pinkfish also prefers the existing formatting convention of 2 spaces per TAB.)

Reply:
Thanks for the answer! I understand you, the code style is an individual thing. I have applied the official code style. I can remove this change. Do I need to revert old code style?

Reply:
I'm not the maintainer but I think generally a PR will get accepted more quickly if it can be reviewed easily. I am not sure that 4 spaces is the "official code style". In Flutter, we generally use 2 spaces although there is no official requirement - each project may have it's own style, this one appears to be 2 spaces.

Reply:
This is the Kotlin code, there are different rules